build.gradle

```diff
@@ -13,7 +13,9 @@ buildscript {
     }
 }
 
+apply plugin: 'base'
 apply from: "${rootProject.buildDir}/release/gradle/version.gradle"
+apply from: "${rootProject.buildDir}/release/gradle/release.gradle"
```

Comment:
You can take a look [here](https://github.com/4finance/micro-infra-spring/blob/master/build.gradle#L22-L41) how multi-module configuration could be handled.